src/store/Network.ts

```diff
@@ -255,6 +275,7 @@ export default {
             commit('setConnected', false);
             commit('connectingToNodeInfo', undefined);
             commit('setFeesConfig', undefined);
+            commit('simpleTransactionDeadline', undefined);
```

Comment:
When reset happen, should reset 
`aggregateTransactionDeadline` and `hashLockTransactionDeadline` as well?